### ISSUE TITLE
Method calling

### ISSUE THREAD

**Issue by idb-** [nccgroup/house#10]
This is not a bug, but more of a feature I was thinking would be useful to House. Perhaps you could implement a way to invoke methods without the necessity of them being hooked prior. Would be a nice addition. :)

**Comment by H4oK3** [comment]
Yeah thats definitely something cool for the tool; will work on it when I get time :)

**Comment by pwnstorm** [comment]
Any update on this feature request, I was thinking it could be implemented in a REST API, this would help calling methods, in an external script. For example, an application I have been looking at implements custom encryption, with this feature one could create a script that decrypts values on the fly.
